ros2topic/ros2topic/api/__init__.py

```diff
@@ -171,6 +187,17 @@ def add_qos_arguments_to_argument_parser(
              .format(verb, default_preset))
     default_profile = rclpy.qos.QoSPresetProfiles.get_from_short_key(
         default_preset)
+    parser.add_argument(
+        '--qos-depth', metavar='N', type=unsigned_int, default = default_profile.depth,
+        help='Queue size setting to {} with '
+             '(overrides depth value of --qos-profile option, default: {})'
+             .format(verb, default_profile.depth))
```

Comment:
The depth here isn't necessarily a default of 0 in the case of transient_local.  Not sure how we want to communicate that, though.

Reply:
we could check the default profile and and if the durability is transient_local AND depth==0, we can show depth=1 instead?

Reply:
I will set -1 as default value, and not printing out.